src/signals/triage_signals.R

```diff
@@ -140,8 +145,9 @@ preview_campaign_urls <- function(campaign_urls) {
 #' @param df Signals data frame
 #' @param fn_signals File name to the signals data
 #' @param test Whether or not the signals were in the test folder.
-approve_signals <- function(df, fn_signals, test) {
+approve_signals <- function(df, fn_signals, test, user_command='init'){
```

Comment:
Make sure to check the linting for style matching. You can run it locally just running `lintr::lint_dir()`. All of the settings we use are in `.lintr`. For instance, always need spaces around arguments, and no single quotes (unless necessary for wrapping double quotes in a string for instance). So, `user_command = "init"`.